lib/jsonapi-response.js

```diff
@@ -0,0 +1,102 @@
+const attributesMap = require('../fixtures/attributes');
+const relationshipsMap = require('../fixtures/relationships');
+const config = require('../config');
+
+module.exports = (data) => {
+  const response = {};
+
+  response.data = {};
+  response.data.type = data._type;
+  response.data.id = data._id;
+  response.data.attributes = getAttributes(data._source);
+
+  response.relationships = getRelationships(data._source);
+
+  response.included = getIncluded(data._source, response);
+
+  response.links = {};
+  response.links.self = config.rootUrl + '/' + data._type + '/' + data._id;
+
+  return JSON.stringify(response);
+};
+
+function getAttributes (data, dataType) {
+  const attributes = {};
+
+  Object.keys(data).forEach((key) => {
+    if (attributesMap.indexOf(key) > -1) {
+      attributes[key] = data[key];
+    }
+  });
+
+  return attributes;
+}
+
+function getRelationships (data) {
+  const relationships = {};
+
+  Object.keys(data).forEach((key) => {
+    if (relationshipsMap.indexOf(key) > -1) {
+      relationships[key] = {data: []};
+
+      data[key].forEach((el, i) => {
+        let rel = {
+          type: '',
+          id: ''
+        };
+
+        rel.id = el.admin.uid;
+        rel.type = el.admin.id.split('-')[0];
+
+        relationships[key].data.push(rel);
+      });
+
+      if (relationships[key].data.length === 1) {
+        relationships[key].data = relationships[key].data[0];
+      }
+
+      if (relationships[key].length === 1) {
+        relationships[key] = relationships[key][0];
+      }
```

Comment:
For this one, it should always be just one object, so I didn't think an array made sense